Client/src/main/java/de/baleipzig/pdfextraction/controller/ImportController.java

```diff
@@ -28,9 +51,63 @@ private void continueButtonOnAction() {
         );
     }
 
-    // initialize wird automatisch vom FXML loader aufgerufen wenn die FXML Datei geladen wird
     @FXML
-    private void initialize(){
-        PDFPreview.createPreview(PDFPreviewPlaceholder);
+    private void initialize() {
+
+        // image view resizeble machen
+        AnchorPane parent = (AnchorPane) imageView.getParent();
+        imageView.fitWidthProperty().bind(parent.widthProperty());
+        imageView.fitHeightProperty().bind(parent.heightProperty());
+
+        if (pdfPath == null){
+            pageIndex.setText("Seitenanzahl");
+        } else {
+            updatePdfPreview(PDFPreview.getCurrentPage());
+        }
+    }
+
+    public void onClickPageBack(ActionEvent actionEvent) {
+
+        if (PDFPreview.getCurrentPage() > 1 && pdfPath != null){
+
+            int pageNumber = PDFPreview.getCurrentPage() - 1;
+            updatePdfPreview(pageNumber);
+        }
     }
+
+    public void onClickPageForward(ActionEvent actionEvent) {
+
+        if (PDFPreview.getCurrentPage() < PDFPreview.numberOfPages && pdfPath != null){
+
+            int pageNumber = PDFPreview.getCurrentPage() + 1;
+            updatePdfPreview(pageNumber);
+        }
+    }
+
+    public void onClickChooseFile(ActionEvent actionEvent) {
+
+        // Filechooser
+        Stage currentStage = (Stage) this.continueButton.getScene().getWindow();
+        FileChooser fileChooser = new FileChooser();
+
+        File selectedFile = fileChooser.showOpenDialog(currentStage);
+        pdfPath = Paths.get(selectedFile.toURI());
+
+        // die aktuelle Seitenzahl soll resetet werden wenn eine neue Datei geladen wird
+        PDFPreview.setCurrentPage(1);
+        updatePdfPreview(PDFPreview.getCurrentPage());
+    }
+
+    private void loadImage(){
+        Image previewImage = PDFPreview.createPreviewImage(PDFPreview.getCurrentPage() - 1, pdfPath);
```

Comment:
Das erscheint mir stark unlogisch, das du immer -1 rechnen musst

Reply:
Das hat den grund, das die lib PDF Box bei 0 anfängt zu zählen, die Seitenzahl in der View aber beginnnend bei 1 angezeigt wird. Ich hab mir halt eine stelle gesucht in der ich das 1 mal anpassen muss, und das war eben diese